src/Apps/Artwork/Components/ArtworkDetails/__tests__/ArtworkDetails.test.tsx

```diff
@@ -135,23 +156,32 @@ describe("ArtworkDetails", () => {
     })
 
     it("renders partner follow button for regular partner with profile", async () => {
-      const data = cloneDeep(ArtworkDetailsFixture)
-      data.partner.type = "NOT Auction House"
-      wrapper = await getWrapper(data)
+      wrapper = await getWrapper({
+        ...ArtworkDetailsFixture,
+        partner: {
+          ...ArtworkDetailsFixture.partner,
+          type: "NOT Auction House",
+        },
+      })
       expect(wrapper.html()).toContain("Following")
     })
 
     it("does not render partner follow button if artwork is from an auction partner", async () => {
-      const data = cloneDeep(ArtworkDetailsFixture)
-      data.partner.type = "Auction House"
-      wrapper = await getWrapper(data)
+      wrapper = await getWrapper({
+        ...ArtworkDetailsFixture,
+        partner: {
+          ...ArtworkDetailsFixture.partner,
+          type: "Auction House",
+        },
+      })
```

Comment:
Why unwrap the `cloneDeep` on the right with additional spreads?

Reply:
The returned type still has readonly modifiers

Reply:
👍

Reply:
Just did a quick test, this would work at runtime:

```ts
      wrapper = await getWrapper(
        deepmerge(ArtworkDetailsFixture, {
          partner: {
            type: "Auction House",
          },
        })
      )
```

but the `deepmerge` typings need some work to work well for us, otherwise we lose all type benefits. We only want [this single overload](https://github.com/TehShrike/deepmerge/blob/7640d50c9d9dcd5a17a6e92fdca1bd6de9d9caf9/index.d.ts#L1) and it should accept a _deep_ `Partial` for `y`.

All doable, of course.

/cc @erikdstock